scapy/contrib/geneve.py

```diff
@@ -1,5 +1,3 @@
-# Copyright (C) 2018 Hao Zheng <haozheng10@gmail.com>
```

Comment:
Can you explain why you are removing this?

Reply:
I checked in few files and such personal declaration was not presento So i remove it. Shall i re add?

Reply:
Yes please. Some files in our source code have this kind of line.